ui/perfherder/compare/CompareTable.jsx

```diff
@@ -27,7 +27,17 @@ export default class CompareTable extends React.Component {
   constructor(props) {
     super(props);
     const { data } = this.props;
-
+    const [firstElementOfData] = data;
```

Comment:
Please extract this into a separate helper method. There's too much code for a constructor.